To double your money in stocks in 5 years, what return would you need?

The Compound Annual Growth Rate (CAGR) is:
$$ (1 + r)^t = 1 + R $$

To double your money in 5 years you would need:
$$ (1 + r)^5 = 1 + 1 = 2 $$

to find r, you need to exponentiate:
$$ r = e^{\frac{ln(1 + R)}{t}} - 1$$

In [1]:
from IPython.core.display import HTML
def css_styling():
    styles = open("./styles/custom.css", "r").read  ()
    return HTML(styles)
css_styling()

%matplotlib inline
import numpy as np
import matplotlib.pyplot as plt
import os
print(os.getcwd())

/Users/rinckd/__Code/__mathCreativity/notebooks


In [4]:
r = np.exp(np.log(1.0 + 1.0)/ 5.0) - 1
print (r)

0.148698354997


0.148 or ~15% is ambitious. The DOW has a long-term CAGR of 8%. 
Apple, from 1990 to 2013 has a 22% CAGR.

In [7]:
r = np.exp(np.log(1.0 + 1.0)/ 20.0) - 1
print(r)

0.0352649238414


Doubling your money in 20 years is more realistic. You would only need 3.5% CAGR